### PR TITLE
More Vests

### PR DESCRIPTION
**Pull request by DarkLevelSS13**
## About The Pull Request
A dark vest made out of light, breathable fabric. Feeling lucky, punk?
image

A dark blue jacket with black highlights. You can't think of any reason why someone would need a windbreaker in space, but the jacket looks cool either way. As an added bonus, it looks fairly resistant to stains and caustic chemicals.
image

A red vest with golden streaks. It's made out of tough materials, and can protect fairly well against bullets. Wake the fuck up, Samurai.
image

## Why It's Good For The Game
Punk maintenance.

## Changelog
:cl: DarkLevel - coding; Armolitskiy - sprites.
add: Punk Vest
add: Windbreaker
add: Cyberpunk vest
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -373,3 +373,19 @@
...
+	armor = list(
+		melee = 10,
+		bullet = 10,
+		energy = 0,
+		bomb = 0,
+		bio = 0,
+		rad = 0
+	)
+	body_parts_covered = UPPER_TORSO
+	flags_inv = HIDEJUMPSUIT
```
Comment (MadmanMartian) [inline code/modules/clothing/suits/miscellaneous.dm:391]: Why would it hide your jumpsuit?

Comment (DarkLevelSS13) [reply code/modules/clothing/suits/miscellaneous.dm:391]: So vest can be on top of jumpsuit.

Comment (MadmanMartian) [reply code/modules/clothing/suits/miscellaneous.dm:391]: Suits are already on top of jumpsuits. Having this flag would just make the jumpsuit invisible while wearing the jacket.

You're making a jacket that makes people naked.

Comment (DarkLevelSS13) [reply code/modules/clothing/suits/miscellaneous.dm:391]: Okay, i will remove this flag.

Comment (DarkLevelSS13) [reply code/modules/clothing/suits/miscellaneous.dm]: Done

**Review by genessee596 — APPROVED** [review]
>What are you looking punk

Engrish is one of the best parts of being a delegate

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -373,3 +373,19 @@
 	cold_protection = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 	min_cold_protection_temperature = T0C - 20
 	siemens_coefficient = 0.7
+
+/obj/item/clothing/suit/punkvest
+	name = "punk vest"
+	desc = "What are you looking punk?"
```
Comment (MadmanMartian) [inline code/modules/clothing/suits/miscellaneous.dm:379]:
```suggestion
	desc = "What are you looking at, punk?"
```

**Comment by ArmolitskiyCarpoeb** [comment]
merge when

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -373,3 +373,49 @@
...
+		energy = 0,
+		bomb = 0,
+		bio = 30,
+		rad = 0
+	)
+	body_parts_covered = UPPER_TORSO|ARMS
+
+/obj/item/clothing/suit/punkvest/cyber
+	name = "cyberpunk vest"
+	desc = "Only the coolest cyberpunks wear this."
```
Comment (genessee596) [inline code/modules/clothing/suits/miscellaneous.dm:411]:
```suggestion
	desc = "A red vest with golden streaks. It's made out of tough materials, and can protect fairly well against bullets. Wake the fuck up, Samurai."
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 4]
```diff
@@ -373,3 +373,49 @@
...
+		energy = 0,
+		bomb = 0,
+		bio = 0,
+		rad = 0
+	)
+	body_parts_covered = UPPER_TORSO
+
+/obj/item/clothing/suit/storage/toggle/windbreaker
+	name = "windbreaker"
+	desc = "The latest TX-corp model to keep you out of the wind and rain. Why, then, is it needed in space?"
```
Comment (genessee596) [inline code/modules/clothing/suits/miscellaneous.dm:394]:
```suggestion
	desc = "A dark blue jacket with black highlights. You can't think of any reason why someone would need a windbreaker in space, but the jacket looks cool either way. As an added bonus, it looks fairly resistant to stains and caustic chemicals."
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 5]
```diff
@@ -373,3 +373,49 @@
 	cold_protection = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 	min_cold_protection_temperature = T0C - 20
 	siemens_coefficient = 0.7
+
+/obj/item/clothing/suit/punkvest
+	name = "punk vest"
+	desc = "What are you looking at, punk?"
```
Comment (genessee596) [inline code/modules/clothing/suits/miscellaneous.dm:379]:
```suggestion
	desc = "A dark vest made out of light, breathable fabric. Feeling lucky, punk?"
```

**Review by TheShown911 — APPROVED** [review]

**Review by TorinoFermic — APPROVED** [review]

**Pull request merged**